### PR TITLE
Jellyfin.JellyfinMediaPlayer version 1.9.1

### LINKED ISSUE #103480: [Package Issue]: Jellyfin.JellyfinMediaPlayer
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

No applicable update found.

### Brief description of your issue

A security update, version 1.9.1 is out now. It should be updated.

### Steps to reproduce

`winget upgrade Jellyfin.JellyfinMediaPlayer -e`

### Actual behavior

`No applicable upgrade found.`

### Expected behavior

`Found JellyfinMediaPlayer [Jellyfin.JellyfinMediaPlayer] Version 1.9.1`

### Environment

```raw
Windows Package Manager v1.4.10173
Windows: Windows.Desktop v10.0.22621.1555
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.10173.0
```

### Screenshots and Logs

_No response_

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
Resolve #103480 

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103685)

### CODE REVIEW

File: manifests/j/Jellyfin/JellyfinMediaPlayer/1.9.1/Jellyfin.JellyfinMediaPlayer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+PackageIdentifier: Jellyfin.JellyfinMediaPlayer
+PackageVersion: 1.9.1
+PackageLocale: en-US
+Publisher: Jellyfin
+PublisherUrl: https://jellyfin.org/
+PublisherSupportUrl: https://jellyfin.org/contact/
+PackageName: Jellyfin Media Player
+PackageUrl: https://github.com/jellyfin/jellyfin-media-player/
+License: GPL-2.0-or-later
```
Comment (mdanish-kh) [inline manifests/j/Jellyfin/JellyfinMediaPlayer/1.9.1/Jellyfin.JellyfinMediaPlayer.locale.en-US.yaml:12]:
```suggestion
License: GPL-2.0
```
I didn't see any info about later versions being supported as a license too in the repo

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
@wingetbot run

Might not matter much but re-running because I see Post-Validation test failed 👀 
image

**Comment by BrandonWanHuanSheng** [comment]
> Might not matter much but re-running because I see Post-Validation test failed 👀 image

It look like Azure Pipelines said "Exception Message: The underlying connection was closed: A connection that was expected to be kept alive was closed by the server. (type WebException)" while you were seeing the pipelines.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**